Prose/ProseLib/Sources/ConversationFeature/Chat/Chat.swift

```diff
@@ -372,6 +442,23 @@ struct ChatView: NSViewRepresentable {
       webView.evaluateJavaScript(script, domain: "Color scheme")
     }
   }
+
+  static func json<T: Encodable & Collection>(_ array: T) -> String {
+    let jsonData: Data
+    do {
+      jsonData = try JSONEncoder().encode(array)
+    } catch {
+      return "[]"
+    }
+    return String(data: jsonData, encoding: .utf8) ?? "[]"
+  }
+
+  /// - Note: This method is named unsafe because it uses force unwraps (`!`).
+  ///         The object is escaped properly.
+  static func unsafeJson<T: Encodable>(_ object: T) -> String {
+    let jsonData: Data = try! JSONEncoder().encode(object)
+    return String(data: jsonData, encoding: .utf8)!
```

Comment:
We could at least crash with `fatalError` and a message that prints the type that was tried to be encoded so that we know what went wrong more easily.